docs/docs/examples-tutorials/recipes/running-tracetest-with-aws-x-ray-adot.md

```diff
@@ -0,0 +1,228 @@
+        condition: service_started
+    healthcheck:
+      test: ["CMD", "wget", "--spider", "localhost:11633"]
+      interval: 1s
+      timeout: 3s
+      retries: 60
+
+  postgres:
+    image: postgres:14
+    environment:
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+    healthcheck:
+      test: pg_isready -U "$$POSTGRES_USER" -d "$$POSTGRES_DB"
+      interval: 1s
+      timeout: 5s
+      retries: 60
+    ports:
+      - 5432:5432
+
+  adot-collector:
+    image: amazon/aws-otel-collector:latest
+    command:
+      - "--config"
+      - "/otel-local-config.yaml"
+    volumes:
+      - ./collector.config.yaml:/otel-local-config.yaml
+    environment:
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+      AWS_SESSION_TOKEN: ${AWS_SESSION_TOKEN}
+      AWS_REGION: ${AWS_REGION}
+    ports:
+      - 4317:4317
+      - 2000:2000
+```
+
+Tracetest depends on Postgres and the ADOT collector. Tracetest requires config files to be loaded via a volume. The volumes are mapped from the root directory into the `root` directory and the respective config files.
+
+The `tracetest.config.yaml` file contains the basic setup of connecting Tracetest to the Postgres instance.
+
+```yaml
+postgres:
+  host: postgres
+  user: postgres
+  password: postgres
+  port: 5432
+  dbname: postgres
+  params: sslmode=disable
+
+poolingConfig:
+  maxWaitTimeForTrace: 10m
+  retryDelay: 5s
+
+googleAnalytics:
+  enabled: true
+```
+
+The `tracetest.provision.yaml` file defines the trace data store, set to the Otel Collector, meaning the traces will be sent to the ADOT instance where later on will be pushed to the AWX X-Ray service and to the OTLP Tracetest endpoint.
+
+But how does Tracetest fetch traces?
+
+The ADOT collector is configured with the `awsxray` receiver and exporting the OpenTelemetry format directly intro Tracetest.
+
+```yaml
+receivers:
+  awsxray:
+    transport: udp
+
+processors:
+  batch:
+
+exporters:
+  awsxray:
+    region: ${AWS_REGION}
+  otlp/tt:
+    endpoint: tracetest:21321
+    tls:
+      insecure: true
+
+service:
+  pipelines:
+    traces/tt:
+      receivers: [awsxray]
+      processors: [batch]
+      exporters: [otlp/tt]
+    traces/xr:
+      receivers: [awsxray]
+      exporters: [awsxray]
+```
+
+How do traces reach AWS X-Ray?
+
+The application code in the `src/index.js` file uses the native AWS SDK X-Ray library which sends telemetry data to the ADOT Collector to be processed and then sent to the configured AWS X-Ray SaaS and Tracetest.
+
+## Run Both the Node.js app and Tracetest
+
+To start both the Node.js app and Tracetest, we will run this command:
+
+```bash
+docker-compose up
+```
+
+This will start your Tracetest instance on `http://localhost:11633/`. Go ahead and open it up.
+Start creating tests! Make sure to use the `http://app:3000/` url in your test creation, because your Node.js app and Tracetest are in the same network.
```

Comment:
```suggestion
Make sure to use the `http://app:3000/` url in your test creation, because your Node.js app and Tracetest are in the same network.
```